library/dd-sdk-android-session-replay/build.gradle.kts

```diff
@@ -72,7 +72,7 @@ android {
     }
 }
 dependencies {
-    implementation(project(":dd-sdk-android"))
```

Comment:
side note: this is something to be reviewed later, but I think we should use `implementation` for our libraries when referencing the core module. This would allow to be more explicit and always require customer to add core module in their setup. But we will discuss it later.

Reply:
yeah need to test it with local maven to see how it behaves with or without